final_prep/README.md

```diff
@@ -32,24 +32,24 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- <img width="889" alt="week 1" src="https://user-images.githubusercontent.com/80938546/115949716-25ba1800-a494-11eb-9c58-2f4870dd7dfa.png">
```

Comment:
Nice job here- I would plan on weekends also filling in with plenty of Turing work. Glad to see you block off times for yourself.